src/mmw/js/src/analyze/templates/table.html

```diff
@@ -2,6 +2,9 @@
     <thead>
         <tr>
             <th data-sortable="true">Type</th>
+            {% if isLandTable %}
+                <th class="hidden-analyze-table-column" data-tableexport-display="always">NLCD Code</th>
```

Comment:
Tried this out & it seemed like there's some quirk getting the `hidden` attribute to take hold via the Nunjucks template.

Going to leave this as is for now, but we should try it again if we end up adjusting these to include more hidden columns.